packages/hardhat-plugin-viem/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@nomicfoundation/hardhat-ignition-viem",
+  "version": "0.11.2",
+  "license": "MIT",
+  "author": "Nomic Foundation",
+  "homepage": "https://hardhat.org",
+  "description": "The Viem extension to Hardhat Ignition. Hardhat Ignition is a declarative system for deploying smart contracts on Ethereum. It enables you to define smart contract instances you want to deploy, and any operation you want to run on them. By taking over the deployment and execution, Hardhat Ignition lets you focus on your project instead of getting caught up in the deployment details.",
+  "repository": "github:NomicFoundation/hardhat-ignition",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "blockchain",
+    "dapps",
+    "tooling",
+    "solidity",
+    "deployment",
+    "viem"
+  ],
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "files": [
+    "dist/src/",
+    "src/",
+    "LICENSE",
+    "README.md",
+    "CHANGELOG.md"
+  ],
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./modules": "./dist/src/modules.js"
+  },
+  "typesVersions": {
+    "*": {
+      "modules": [
+        "./dist/src/modules.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "prebuild": "ts-node ./scripts/compile-test-fixture-projects.ts",
+    "build": "tsc --build",
+    "lint": "npm run prettier -- --check && npm run eslint",
+    "lint:fix": "npm run prettier -- --write && npm run eslint -- --fix",
+    "eslint": "eslint \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\"",
+    "prettier": "prettier \"**/*.{js,ts,md,json}\"",
+    "test": "mocha --recursive \"test/**/*.ts\"",
+    "test:debug": "DEBUG='hardhat-ignition-viem:*,ignition:*' npm run test",
+    "test:coverage": "nyc mocha \"test/**/*.ts\"",
+    "test:build": "tsc --project ./test/",
+    "clean": "rimraf .nyc_output coverage dist tsconfig.tsbuildinfo ./test/fixture-projects/**/deployments ./test/fixture-projects/**/artifacts",
+    "prepack": "npm run build"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-viem": "^1.0.0",
+    "@types/chai": "^4.2.22",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/sinon": "^10.0.13"
+  },
+  "peerDependencies": {
+    "@nomicfoundation/hardhat-viem": "^1.0.0",
+    "hardhat": "^2.18.0"
+  },
+  "dependencies": {
+    "@nomicfoundation/hardhat-ignition": "^0.11.2",
```

Comment:
This should be a peer dependency

Reply:
I have changed both the viem and ethers packages to have peer dependencies instead: https://github.com/NomicFoundation/hardhat-ignition/pull/612/commits/c395a4a4dfdd24438781efe4effbfa8f0a74ba5e